locksmith/src/operations/metadataOperations.ts

```diff
@@ -112,7 +112,7 @@ const defaultMappings = (address: string) => {
   let defaultResponse = {
     name: 'Unlock Key',
     description: 'A Key to an Unlock lock.',
-    image: `${baseURIFragement}/unlock-default-key-image.png`,
+    image: `${config.locksmithUri}/lock/${address}/icon`,
```

Comment:
I think we might be able to do this a bit differently, since the SVG code provides static code regardless of where is coming from and we are returning this relative to a request being made lets see if we can construct this based on information received at a higher level around the host the request is being made to.

This should help to eliminate the need for a constant referring back to the instance serving this request

Reply:
Tell me more? I am not sure I understand?